plugins/rest/plugin_impl_rest.go

```diff
@@ -146,18 +178,15 @@ func (plugin *Plugin) AfterInit() (err error) {
 	plugin.registerAccessListHandlers()
 	plugin.registerInterfaceHandlers()
 	plugin.registerBfdHandlers()
+	plugin.registerNatHandlers()
+	plugin.registerStnHandlers()
+	plugin.registerIPSecHandlers()
 	plugin.registerL2Handlers()
 	plugin.registerL3Handlers()
-
-	plugin.HTTPHandlers.RegisterHTTPHandler("/arps", plugin.arpGetHandler, "GET")
-	plugin.HTTPHandlers.RegisterHTTPHandler(fmt.Sprintf("/acl/interface/{%s:[0-9]+}", swIndexVarName),
-		plugin.interfaceACLGetHandler, "GET")
-	plugin.HTTPHandlers.RegisterHTTPHandler("/command", plugin.commandHandler, "POST")
-	plugin.HTTPHandlers.RegisterHTTPHandler("/telemetry", plugin.telemetryHandler, "GET")
-	plugin.HTTPHandlers.RegisterHTTPHandler("/telemetry/memory", plugin.telemetryMemoryHandler, "GET")
-	plugin.HTTPHandlers.RegisterHTTPHandler("/telemetry/runtime", plugin.telemetryRuntimeHandler, "GET")
-	plugin.HTTPHandlers.RegisterHTTPHandler("/telemetry/nodecount", plugin.telemetryNodeCountHandler, "GET")
-	plugin.HTTPHandlers.RegisterHTTPHandler("/", plugin.indexHandler, "GET")
```

Comment:
We should probably keep the index page at `/` to provide starting point. Perhaps we could make some groups to separate parts related to VPP dumps, Telemetry, etc..

Reply:
I agree, let's keep `/` then.